nethserver-dc.spec

```diff
@@ -1,3 +1,13 @@
+# Do not try to package empty debug_package
+%global debug_package %{nil}
+
+# Naming of 32bit arm is a mess..
+%ifarch %{arm}
+%define source1_arch %{__isa_name}
+%else
```

Comment:
Have second thoughts about the expansion of `%{__isa_name}` after reading the evebox threat on the community forum. And decided to hard-code the arm arch

`%define source1_arch armv7hl`

Reply:
We can go with it, until we decide `aarch64` is interesting too!

Reply:
On aarch64 `%define source1_arch %{_arch}` expands just fine to `aarch64` , So we are good to go here too

Reply:
Please tell me if I understand correctly. Should be changed to:

```spec
# Naming of 32bit arm is a mess..
%ifarch armv7hl
%define source1_arch armv7hl
%else
%define source1_arch %{_arch}
%endif
```

Reply:
yes, this work's for all arch's